src/pages/local-grants/colombia/es.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Subvenciones Locales de Colombia
+description: Una línea de subvenciones de la Fundación Ethereum, específica para Colombia!
```

Comment:
@gilbertginsberg please review this title & description 😄

Reply:
Let's go with: `Programa de Fondos Locales para Colombia`

Reply:
Description is solid